docgen/src/guides/migrate-instantsearchcss.md

```diff
@@ -17,15 +17,17 @@ in v3.
 
 ### Renamed widgets
 
-| In V2 | In V3 |
-| ----- | ----- |
+| In V2    | In V3            |
+| -------- | ---------------- |
+| clearAll | clearRefinements |
 
+### Renamed connectors
 
 ### Renamed parameters
 
-| In V2 | In V3 |
-| ----- | ----- |
-
+| In V2           | In V3                   |
+| --------------- | ----------------------- |
+| connectClearAll | connectClearRefinements |
```

Comment:
It should be in the section above "Renamed connectors" rather than "Renamed parameters".